apps/rush-lib/src/cli/actions/BaseRushAction.ts

```diff
@@ -69,7 +76,11 @@ export abstract class BaseConfiglessRushAction extends CommandLineAction {
       }
     }
 
-    if (!Utilities.shouldRestrictConsoleOutput()) {
+    if (!this._suppressStartupBanner && !Utilities.shouldRestrictConsoleOutput()) {
+      const banner: string | undefined = this._parser.startupBanner;
+      if (banner) {
+        console.log(banner);
+      }
       console.log(`Starting "rush ${this.actionName}"${os.EOL}`);
```

Comment:
Printing this kinda goes against the "ensure the action has full control over logging output" premise of this feature.

Reply:
That's why this is inside the `if (!this._suppressStartupBanner)` conditional?